tests/init.test.js

```diff
@@ -1,49 +1,103 @@
 const utils = require('./utils');
-const testRepo = require('./utils').testRepo;
 const fs = require('fs-extra');
 const path = require('path');
 const yaml = require('js-yaml');
 
-const { EXPLAIN_PLACEHOLDER } = require('../utils/common');
+const { EXPLAIN_PLACEHOLDER, shouldBeCollapsed } = require('../utils/common');
 
 // Tmp directories used in tests.
 let tmpDirs = Array();
 
 describe('tuture init', () => {
 
   afterAll(() => {
-    tmpDirs.map(dir => fs.removeSync(dir));
+    tmpDirs.forEach(dir => fs.removeSync(dir));
     process.chdir(path.join(__dirname, '..'));
   });
 
-  // TODO: Test prompts when no args given.
+  describe('no .gitignore', () => {
+    // NOTE: When contriving testRepo, put `files` in alphabetic order.
+    const testRepo = [
+      {
+        message: 'Commit 1',
+        files: ['test1.js', 'test2.js'],
+      },
+      {
+        message: 'Commit 2',
+        files: ['package-lock.json'],
+      },
+    ];
+    testInit(testRepo);
+  });
 
-  describe('-y', () => {
-    testInit('-y');
+  describe('.gitignore without ignoring .tuture', () => {
+    const testRepo = [
+      {
+        message: 'Test',
+        files: ['.gitignore', 'app.js'],
+      },
+      {
+        message: 'Another Test',
+        files: ['dir/test1.js', 'dir/yarn.lock'],
+      },
+      {
+        message: 'Still Another Test',
+        files: ['dir/test2.js'],
+      },
+    ];
+    testInit(testRepo);
   });
 
-  describe('--yes', () => {
-    testInit('--yes');
+  describe('.gitignore already having .tuture ignored', () => {
+    const testRepo = [
+      {
+        message: 'First Commit',
+        files: ['.gitignore', 'test1.js'],
+      },
+      {
+        message: 'Second Commit',
+        files: ['test2.js'],
+      },
+    ];
+    testInit(testRepo, true);
+  });
+
+  describe('outside a git repo', () => {
```

Comment:
I'd like naming like this: 

- Git repo exists: `test content`

- Git repo not exists: `test content`

  eg: `test content`
  - no `.gitignore`
  - .gitignore already having .tuture ignored

Reply:
Also, I think we'd better move the only `outside a git repo` **describe block** before all other `Git repo exists` **describe block**.